extensions/VisualEditor/wikia/modules/ve/ve.track.js

```diff
@@ -175,6 +176,18 @@ require( ['wikia.tracker'], function ( tracker ) {
 			return;
 		}
 
+		if ( ve.init.wikia.getToolbarABTestVariantNumber() === 1 && data.label === 'edit-page-ready' ) {
```

Comment:
Why do we need to send extra event to GA in case of the experiment? From what I recall information about the experiment is injected into custom variables anyways?

1. If the information about the experiment is injected into custom variable then there is no need to send extra event.
2. If the information about the experiment is NOT injected into custom variable then we should "duplicate" not only this event but many others (to find out if moving video/image to different location is positive).

I hope it's 1.